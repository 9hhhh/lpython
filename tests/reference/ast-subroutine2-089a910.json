{
    "basename": "ast-subroutine2-089a910",
    "cmd": "lfortran --show-ast {infile} -o {outfile}",
    "infile": "tests/subroutine2.f90",
    "infile_hash": "e9fd527d51cbd1c8733dbb9a1f11be9be90ff8008fdc2a4551cf4419",
    "outfile": "ast-subroutine2-089a910.out",
    "outfile_hash": "25fe753568db8b7170de0613324f2537dba004ce4165e8153dbb32c1",
    "stdout": null,
    "stdout_hash": null,
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}