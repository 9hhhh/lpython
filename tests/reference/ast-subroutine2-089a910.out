(sub g [] [] [] [(= x y) (= x (* 2 y))] [])
